### PR TITLE
Style

### PR DESCRIPTION
**Pull request by ashishprodev**
feat:Add swipe left

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+import TaskManager from './Component/Pages/TaskManager'
+import Todolist from './Component/Pages/Todolist'
+
+
 
 const App = () => {
   return (
-    <View  >
+    <>
+
```
Comment (hangingPandaReact) [inline App.js:12]: Format issue

**Comment by ashishprodev** [comment]
Refactor:files str changes

**Comment by ashishprodev** [comment]
done

File: src/component/Header.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      setTask(taskCopy);
+      setAdd('');
+    }
+  };
+  const leftTask = task.length;
+  return (
+    <SafeAreaView>
+      <View style={styles.mainContainer}>
+        <View>
+          <Image style={styles.img} source={require('../assets/avator.png')} />
```
Comment (hangingPandaReact) [inline src/component/Header.js:30]: make const for this image path

File: src/component/Todolist.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+const Todolist = ({task}) => {
+  const closeRow = (rowMap, rowKey) => {
+    if (rowMap[rowKey]) {
+      rowMap[rowKey].closeRow();
+    }
+  };
+
+  const deleteRow = (rowMap, rowKey) => {
+    closeRow(rowMap, rowKey);
+    console.log({rowKey}, {rowMap});
```
Comment (hangingPandaReact) [inline src/component/Todolist.js:15]: remove console

**Pull request merged**